lighthouse-cli/bin.ts

```diff
@@ -229,11 +235,32 @@ function lighthouseRun(addresses: Array<string>, config: Object, flags: Object)
   return lighthouse(address, flags, config)
     .then((results: Results) => Printer.write(results, outputMode, outputPath))
     .then((results: Results) => {
+      const filename = `${assetSaver.getFilenamePrefix({url: address})}.report.html`;
+
       if (outputMode === Printer.OutputMode[Printer.OutputMode.pretty]) {
-        const filename = `./${assetSaver.getFilenamePrefix({url: address})}.report.html`;
         Printer.write(results, 'html', filename);
       }
 
+      // Generate report.html, host it and open it in the default browser
+      if (flags.view) {
+        server.getPort(0).then((port: number) => {
+          const filePath = `${server.FOLDERS.REPORTS}/${filename}`;
+          const htmlExist = outputMode === Printer.OutputMode[Printer.OutputMode.pretty]
```

Comment:
Would it be better to just check if the file exists to avoid having two places to change any logic on when a file will be printed to file?